.gitignore

```diff
@@ -3,3 +3,5 @@
 /.idea
 Cargo.lock
 /target
+bindings/java/java/src/main/resources
+bindings/java/java/src/test/resources
```

Comment:
These should go into `bindings/java/.gitignore`.

Reply:
done